docs/docs/modules/indexes/vector_stores/integrations/tigris.md

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
Can you change this to a `.mdx` file and move the code blocks to `/examples`?

You can read more about it here:

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#documentation-and-integration-tests